lib/src/Pos/Launcher/Resource.hs

```diff
@@ -172,7 +172,7 @@ releaseNodeResources ::
        NodeResources ext -> Production ()
 releaseNodeResources NodeResources {..} = do
     whenJust nrJLogHandle (liftIO . hClose)
-    closeNodeDBs nrDBs
+    liftIO $ closeNodeDBs nrDBs
```

Comment:
Why was the signature for `closeNodeDBs` changed to `IO` instead of `MonadIO m => m`?